website/api/views.py

```
@@ -1031,3 +1034,200 @@ def post(self, request, *args, **kwargs):
 
         return Response(report, status=status.HTTP_200_OK)
+
+
+class JobViewSet(viewsets.ModelViewSet):
+    """
+    API ViewSet for Job CRUD operations
+    Requires authentication for create, update, delete
+    """
+
+    serializer_class = JobSerializer
+    permission_classes = [IsAuthenticatedOrReadOnly]
```

Ruff (0.14.5)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)